tests/unit/s2n_config_test.c

```diff
@@ -1086,5 +1087,97 @@ int main(int argc, char **argv)
         }
     }
 
+    /* Test s2n_config_validate_certificate_preferences */
+    {
+        DEFER_CLEANUP(struct s2n_cert_chain_and_key *ecdsa_p384_sha256 = NULL,
+                s2n_cert_chain_and_key_ptr_free);
+        DEFER_CLEANUP(struct s2n_cert_chain_and_key *ecdsa_p384_sha384 = NULL,
+                s2n_cert_chain_and_key_ptr_free);
+        EXPECT_SUCCESS(s2n_test_cert_chain_and_key_new(&ecdsa_p384_sha256, S2N_KITTEN_SAN_CHAIN,
+                S2N_KITTEN_SAN_KEY));
+        EXPECT_SUCCESS(s2n_test_cert_permutation_load_server_chain(&ecdsa_p384_sha384, "ec",
+                "ecdsa", "p384", "sha384"));
+
+        /* rfc9151 doesn't allow SHA256 signatures, but does allow SHA384 signatures, 
+         * so ecdsa_p384_sha256 is invalid and ecdsa_p384_sha384 is valid */
+
+        /* valid certs are accepted */
+        {
+            DEFER_CLEANUP(struct s2n_config *config = s2n_config_new(), s2n_config_ptr_free);
+            EXPECT_SUCCESS(s2n_config_add_cert_chain_and_key_to_store(config, ecdsa_p384_sha384));
+            EXPECT_OK(
+                    s2n_config_validate_certificate_preferences(config, &security_policy_rfc9151));
```

Comment:
haha it seems like this indent behavior is why we don't enforce a column limit in CI:
https://github.com/aws/s2n-tls/pull/3633#discussion_r1028544134

The clang format intending is definitely strange, so I would vote manually formatting. But if it doesn't bother anyone else then I'm fine with it.

Reply:
Our coding convention is to just keep everything on the same line until it gets longer than like 100-120ish characters and then move the rest to the next line. James, can you fiddle with your clang formatter until it resembles the rest of our code? Otherwise you're going to keep getting comments like "there's something off with this indentation."